packages/sdk/src/subgraph/subgraph.ts

```diff
@@ -329,12 +346,11 @@ export class Subgraph {
 
     // Step 2: handle any not-listed active transactions (i.e. sdk has
     // gone offline at some point during the transactions)
-    const errors: Map<string, Error> = new Map();
+    const errors: Map<number, Error> = new Map();
     const txs = await Promise.all(
-      Object.keys(this.sdks).map(async (c) => {
+      inSyncChains.map(async (chainId) => {
```

Comment:
this logic may need to change for the same reasoning as the above comment

Reply:
Gotcha, I think I'll wait for @rhlsthrm's feedback on this one.

Reply:
Agreed, we need to return all the active transactions regardless of out of sync. In fact on the SDK I don't think we need to care about out-of-sync at all in this subgraph loop. We only need to:
1. Not send auction requests involving a particular chain if all subgraphs are out of sync.
2. Choose the "in sync" subgraph out of all the fallbacks.
3. Once we have the subgraph chosen, just treat it as in-sync.

What are the cases we need to worry about on the SDK?
- Sender stops syncing
  - In this case, we won't see a sender prepare on the active transactions. That's fine, just continue to monitor and poll and make sure we are using the synced subgraph (if any) on each loop iteration.
- Receiver stops syncing
  - Will not see the receiver prepare after the router does it. That's fine, just continue to monitor and poll and make sure we are using the synced subgraph (if any) on each loop iteration.

We don't actually need to change any of the active tx logic here, all this needs to do is choose the correct subgraph.

This might be true for the router as well. I think this only needs to affect the initial auction methods.